src/m365/entra/commands/m365group/m365group-user-list.ts

```diff
@@ -127,7 +123,7 @@ class EntraM365GroupUserListCommand extends GraphCommand {
         owners.forEach(owner => users.push({ ...owner, roles: ['Owner'], userType: 'Owner' }));
```

Comment:
@nanddeepn based on what is described in the issue and @milanholemans I think we are not done yet with this change. 
We should not overwrite no use the `userType` property and leave it as is. It should be either `Member` of `Guest`.
Now we modify it by role and as a result my user has `owner` in this property which is not valid.
![image](https://github.com/user-attachments/assets/0091439b-53e8-4133-b952-a359b83b9009)

Reply:
Yes, that is correct. We should remove all `userType` code and leave it untouched.